### PR TITLE
PE-3977: Release v2.2.0

### PR DESCRIPTION
**Pull request by matibat**
Suggested release notes
- Removes the “Insufficient Funds“ warning underneath the New button

### CODE REVIEW

**Comment by matibat** [comment]
This PR is carrying unexpected changes from dev. Will be on hold til I get instructions on how to proceed.

File: assets/config/prod.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
   "defaultArweaveGatewayUrl": "https://arweave.net",
-  "useTurbo": true,
-  "defaultTurboUrl": "https://upload.ardrive.io",
+  "useTurboUpload": true,
+  "useTurboPayment": false,
```
Comment (matibat) [inline assets/config/prod.json:4]: Disabled at prod 👍

File: pubspec.yaml
Comment (matibat) [inline pubspec.yaml:1]: Ardrive Http, Ardrive IO and ArDrive UI are holding a temporal version, are we OK with that?

**Comment by matibat** [comment]
The last commit broke the build

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM!

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM!

**Pull request merged**